modules/nlu/src/backend/module-lifecycle/on-server-started.ts

```diff
@@ -125,13 +125,13 @@ const registerMiddleware = async (bp: typeof sdk, state: NLUState) => {
     }
   })
 
-  function removeSensitiveText(event) {
+  function removeSensitiveText(event: sdk.IO.IncomingEvent) {
     if (!event.nlu.entities || !event.payload.text) {
       return
     }
 
     try {
-      const sensitiveEntities = event.nlu.entities.filter(ent => ent.sensitive)
```

Comment:
`ent.sensitive` was always undefined so the result list was always empty?

Reply:
yes ... not only for list entities but for all entities. this is a regression we introduced in 12.0.0